scripts/circleci/setup-angular-snapshots.js

```diff
@@ -36,6 +36,13 @@ console.log(`  ${angularPackages.map(n => `${n}${packageSuffix}`).join('\n  ')}\
 
 // Setup the snapshot version for each Angular package specified in the "package.json" file.
 angularPackages.forEach(packageName => {
+  // Skip update for the shared dev-infra package. We do not want to update to a snapshot
+  // version of the dev-infra tooling as that could break tooling from running snapshot
+  // tests for the actual snapshot Angular framework code.
+  if (packageName === '@angular/dev-infra-private') {
```

Comment:
nit: we might want to have a set of packages that we shouldn't update at the top of the file.